static-website-aws-csharp/Program.cs

```diff
@@ -0,0 +1,114 @@
+using System.Collections.Generic;
+using Pulumi;
+using Aws = Pulumi.Aws;
+using SyncedFolder = Pulumi.SyncedFolder;
+
+return await Deployment.RunAsync(() =>
+{
+    // Import the program's configuration settings.
+    var config = new Config();
+    var path = config.Get("path") ?? "./site";
+    var indexDocument = config.Get("indexDocument") ?? "index.html";
+    var errorDocument = config.Get("errorDocument") ?? "error.html";
+
+    // Create an S3 bucket and configure it as a website.
+    var bucket = new Aws.S3.Bucket("bucket", new()
+    {
+        Acl = "public-read",
```

Comment:
I'll run through and add some enums as well for the languages that have them for these.

Reply:
Actually on second thought, I'll do this as a follow-up, as they'll need to be added to Synced Folder as well, and I don't want to block on that. Easy enough to add post-release.